### DISCUSSION TITLE
Separate multiple Speakers (vocals) similar to Meta's model ?

### DISCUSSION THREAD

**Discussion by AlonDan** [Anjok07/ultimatevocalremovergui#1104] [General]
UVR already can do lots of separation such as vocals, music, de-reverb, de-echo, etc..

Is there a MODEL that allow to separate multiple speakers? for example:
Sometimes in Podcast 2 or more people are talking together, separating each human voice could be nice.

I believe Meta have something related, you can read more about it here:
[voice separation model that distinguishes multiple speakers simultaneously](https://ai.meta.com/blog/a-new-state-of-the-art-voice-separation-model-that-distinguishes-multiple-speakers-simultaneously/)
Since I'm not sure if there is already such model for UVR I decided to ask in here.

Maybe there is already such MODEL to download but if so, I missed it.
If not... it will be amazing to have this option in the future to separate multiple speakers.

Please share / update in here if there are news on this subject, thanks ahead! 🙏

**Comment by Benji0TheFox** [comment]
It'd be kinda cool to find a way to separate background music from people talking. Especially when there's singing in some songs, it makes it questionable at times whether if that can be done.

**Reply by antonskal** [reply]
Demucs exists which can do this, I think...

**Comment by SureshKumarKummari** [comment]
Hey any one know answer for this?

**Comment by stefan-sherwood** [comment]
This is not an answer to your question per se but it is germane to the problem you're trying to solve

https://huggingface.co/pyannote

**Comment by Vrezerino** [comment]
I'm also interested in this.
